src/App/Pages/Vault/AddEditPageViewModel.cs

```diff
@@ -455,6 +455,7 @@ public async Task<bool> SubmitAsync()
                 _platformUtilsService.ShowToast("success", null,
                     EditMode && !CloneMode ? AppResources.ItemUpdated : AppResources.NewItemCreated);
                 _messagingService.Send(EditMode && !CloneMode ? "editedCipher" : "addedCipher", Cipher.Id);
+               await _deviceActionService.RefreshAutoFillAsync();
```

Comment:
formatting (need one more space before `await`, unless github is just rendering funny)

Reply:
Fixed.